build.gradle

```diff
@@ -4,115 +4,128 @@ buildscript {
-  }
-}
+group = "com.icthh.xm.ms.configuration"
+version = "0.0.1-SNAPSHOT"
 
-defaultTasks 'bootRun'
+description = ""
 
-group = 'com.icthh.xm.ms.configuration'
-version = '0.0.1-SNAPSHOT'
+sourceCompatibility=11
+targetCompatibility=11
+assert System.properties["java.specification.version"] == "11" || "12" || "13"
 
-description = ''
+apply from: "gradle/docker.gradle"
+apply from: "gradle/sonar.gradle"
+//jhipster-needle-gradle-apply-from - JHipster will add additional gradle scripts to be applied here
 
-bootWar {
-   mainClassName = 'com.icthh.xm.ms.configuration.ConfigurationApp'
+if (project.hasProperty("prod")) {
+    apply from: "gradle/profile_prod.gradle"
+} else {
+    apply from: "gradle/profile_dev.gradle"
 }
 
-war {
+if (project.hasProperty("war")) {
+    apply from: "gradle/war.gradle"
+}
 
+if (project.hasProperty("zipkin")) {
+    apply from: "gradle/zipkin.gradle"
 }
 
-springBoot {
-    mainClassName = 'com.icthh.xm.ms.configuration.ConfigurationApp'
-    buildInfo()
+idea {
+    module {
+        excludeDirs += files("node_modules")
+    }
 }
 
+eclipse {
+    sourceSets {
+        main {
+            java {
+                srcDirs += ["build/generated/sources/annotationProcessor/java/main"]
+            }
+        }
+    }
+}
+
+// See https://virgo47.wordpress.com/2018/09/14/classpath-too-long-with-spring-boot-and-gradle/ for details
+// https://github.com/jhipster/generator-jhipster/issues/9713
 if (OperatingSystem.current().isWindows()) {
-    // https://stackoverflow.com/questions/40037487/the-filename-or-extension-is-too-long-error-using-gradle
     task classpathJar(type: Jar) {
-        dependsOn configurations.runtime
-        appendix = 'classpath'
+        inputs.files sourceSets.main.runtimeClasspath
 
+        archiveFileName = "runboot-classpath.jar"
         doFirst {
             manifest {
-                attributes 'Class-Path': configurations.runtime.files.collect {
-                    it.toURI().toURL().toString().replaceFirst(/file:\/+/, '/').replaceAll(' ', '%20')
-                }.join(' ')
+                def classpath = sourceSets.main.runtimeClasspath.files
+                attributes "Class-Path": classpath.collect {f -> f.toURI().toString()}.join(" ")
             }
         }
     }
 
     bootRun {
-        dependsOn classpathJar
-        doFirst {
-            classpath = files("$buildDir/classes/java/main", "$buildDir/resources/main", classpathJar.archivePath)
-        }
+        classpath = classpathJar.outputs.files
     }
 }
 
+bootWar {
+    mainClassName = 'com.icthh.xm.ms.configuration.ConfigurationApp'
+}
+
+war {
+
+}
+
+defaultTasks "bootWar"
+
+springBoot {
+    mainClassName = "com.icthh.xm.ms.configuration.ConfigurationApp"
+}
+
 test {
-    systemProperty "spring.profiles.active", "test"
-    mkdir("src/test/features")
-    include '**/*UnitTest*'
-    include '**/*MvcTest*'
-    include '**/*IntTest*'
```

Comment:
Let's copy this includes to new versionn

Reply:
done